<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OwnerHome</class>
 <widget class="QWidget" name="OwnerHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>业主首页</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小区业主信息平台</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重要信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="duePaymentLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>待缴费用：</string>
   </property>
  </widget>
  <widget class="QLabel" name="duePaymentValue">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0元</string>
   </property>
  </widget>
  <widget class="QLabel" name="maintenanceLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我的维修单：</string>
   </property>
  </widget>
  <widget class="QLabel" name="maintenanceValue">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0单</string>
   </property>
  </widget>
  <widget class="QLabel" name="noticeTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小区公告</string>
   </property>
  </widget>
  <widget class="QListWidget" name="noticeList">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="noticeContentTitle">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>公告详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectedNoticeTitle">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>150</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择公告查看详情</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="noticeContent">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>200</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="publishTimeLabel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>720</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
